rest/data/aliases.go

```diff
@@ -126,8 +126,8 @@ func updateAliasesForSection(projectId string, updatedAliases []restModel.APIPro
 
 // validateFeaturesHaveAliases returns an error if project/repo aliases are not defined for a Github/CQ feature.
 // Does not error if version control is enabled.
-func validateFeaturesHaveAliases(pRef *model.ProjectRef, aliases []restModel.APIProjectAlias) error {
-	if pRef.IsVersionControlEnabled() {
+func validateFeaturesHaveAliases(originalProjectRef *model.ProjectRef, newProjectRef *model.ProjectRef, aliases []restModel.APIProjectAlias) error {
+	if originalProjectRef.IsVersionControlEnabled() {
```

Comment:
Should we add a comment here or at 128 about why we use this other project ref?

Reply:
Sure, adding